### ISSUE TITLE
CommandLine & Json Config

### ISSUE THREAD

**Issue by tiayi** [aspnet/Extensions#767]
How can I use command line arguments override JSON array config section?
I tried as below, but it doesn't work well.

JSON
`{ "names": ["a","b","c","d"] }`
Command args 
`app.exe -names d,e,f `

Code `configuration["names"]` can get right value from command args.
Code `configuration['names'].GetChildren().Select(p=>p.Value)`  returns `"a","b","c","d"`

Could anyone help me?

**Comment by ajcvickers** [comment]
I don't believe that this is something that is possible right now. Adding to the backlog to consider later.

**Comment by anurse** [comment]
This is part of a broader category of issues related to array representation in Config. It's not likely to be something we can change significantly without a major redesign. Transferring to the appropriate repo and putting in the Future milestone though.
